FanSpeedControl/FanSpeedControl.module.less

```diff
@@ -0,0 +1,46 @@
+// FanSpeedControl.module.less
+//
+
+@import '../styles/mixins.less';
+@import '../styles/skin.less';
+@import '../styles/variables.less';
+
+.fanSpeedControl {
+	display: inline-flex;
+	position: relative;
+
+	.fanSpeedControlSvg {
+		height: 300px;
+		width: 300px;
```

Comment:
In the GUI guide, there is 240px x 240px fan control. We might support `size` prop for different size control.

Reply:
This one: https://app.zeplin.io/project/5eb9f7179bdee84a932d95ef/screen/5f154437e54add7dc62947ca ?
Do you want me to create "small" and "large" sizes? (240px / 300px)